parm/config/gfs/config.ufs

```diff
@@ -430,4 +466,44 @@ if [[ "${skip_ww3}" == "false" ]]; then
   export ntasks_ww3 nthreads_ww3
 fi
 
+# GOCART specific settings
+if [[ "${DO_AERO}" = "YES" ]]; then # temporary settings for aerosol coupling
```

Comment:
We have the `skip_` thingy going on here.  Would we want to stay with that for aerosols and define `skip_gocart` and use that toggle?

Reply:
This is going to require more work, so I'm going to vote it as out-of-scope. To eliminate the need for the `DO_*` switches:
- The argparsing needs to be updated, as there wouldn't be an argument for `--gocart`, so should only `shift` once
- Need to redo to the component config loading on L64-69, as the config files are all by component but the skip switches are all based on model name

Reply:
I don't think we can use the `shift` thing here because `DO_AERO` makes sense if there is an atmosphere.
The inputs to `config.ufs` are `--fv3 $fv3_res --mom6 $mom6_res` etc.  There is not going to be a `--gocart $gocart_res` as GOCART runs on the same grid as atmosphere (FV3).
The suggestion was to add a default `skip_gocart=true` and then set `skip_gocart=false` if `DO_AERO = YES`.
But this is fine.